en/mozillavpn.xliff

```diff
@@ -168,6 +168,9 @@
         <source xml:space="preserve">Gbps</source>
         <note annotates="source" from="developer">Gigabits per Second</note>
       </trans-unit>
+      <trans-unit id="vpn.connectionInfo.accessibleName">
+        <source xml:space="preserve">%1: %2 %3</source>
```

Comment:
This string requires a comment. Already commented in https://github.com/mozilla-mobile/mozilla-vpn-client/pull/275